doc/configuration.rst

```diff
@@ -382,6 +382,29 @@ release
   "cdh4".
 
 
+[k8s]
```

Comment:
Both k8s and Kubernetes are mutually intelligible to Kubernetes users, but I guess that would be better for continuity. 

What kind of naming convention would you suggest for files though? Should the examples and code be called `kubernetes_job.py` and the test `kubernetes_job_test.py`, or does that not matter as much?